src/harness/reference_models/geo/zones.py

```diff
@@ -389,8 +395,17 @@ def _LoadDpaZones(kml_path, properties):
   # Final code should raise an exception for those which are mandatory by the spec,
   # and use the standard default for the optional ones.
   for name, zone in dpa_zones.items():
+    # CatA neighborhood specified with default value if not in file,
+    # so this is managed in the declaration.
+    # Others seems mandatory:
+    # CatB not yet defined set as NaN
     if np.isnan(zone.catBNeighborhoodDistanceKm):
-      zone.catBNeighborhoodDistanceKm = 200.
+      zone.catBNeighborhoodDistanceKm = 200
+    # OOB distances not yet provided in the KML files, so default to NaN
+    if np.isnan(zone.catAOOBNeighborhoodDistanceKm):
+      zone.catAOOBNeighborhoodDistanceKm = 0
+    if np.isnan(zone.catBOOBNeighborhoodDistanceKm):
+      zone.catBOOBNeighborhoodDistanceKm = 25
 
   return dpa_zones
 
```

Comment:
I think the following changes need to be made in the zones_test.py as well:
- Lines 56 and 68: change "catbNeighborDist" to "catBNeighborhoodDistanceKm".
- Line 68: change "200" to "240".

Reply:
You are right. Thanks! Forgot to run and update that unit test.